index.js

```diff
@@ -95,10 +96,10 @@ function evaluateExponentialFunction(parameters, input) {
     }
 
     if (i === 0) {
-        return parameters.stops[i][1];
+        return overflow ? calculateOverflow(input, base, parameters.stops[i], parameters.stops[i + 1]) : parameters.stops[i][1];
 
     } else if (i === parameters.stops.length) {
-        return parameters.stops[i - 1][1];
+        return overflow ? calculateOverflow(input, base, parameters.stops[i - 1], parameters.stops[i - 2]) : parameters.stops[i - 1][1];
```

Comment:
Can we use still use `interpolate` in this case? Nothing within `interpolate` requires that `input` is within `inputLower` and `inputUpper`. 

If not, `calculateOverflow` will need to handle arrays, like `interpolate`